src/logger/index.js

```diff
@@ -51,7 +62,6 @@ if (config.get('@opentermsarchive/engine.logger.sendMailOnError')) {
       ssl: true,
       timeout: 30 * 1000,
       formatter: args => args[Object.getOwnPropertySymbols(args)[1]], // Returns the full error message, the same visible in the console. It is referenced in the argument object with a Symbol of which we do not have the reference but we know it is the second one.
-      exitOnError: true,
```

Comment:
Why didn't we simply implement the whole thing by removing this line? 🤔 😅

Reply:
If we remove this line while keeping `rejectionHandlers` defined, Winston will catch all `uncaughtException` events without exiting the process, which is not the intended behavior. By default, both Node.js and Winston terminate the process when an `uncaughtException` occurs, as these are considered programmer errors. And this is the correct approach. As explained [here](https://github.com/OpenTermsArchive/engine/blob/main/CONTRIBUTING.md#handling-programmer-errors), crashing immediately is the best way to handle programmer errors.